bindings/cs/MongoDB.Crypt/AwsKeyId.cs

```diff
@@ -67,8 +87,13 @@ void IInternalKmsKeyId.SetCredentials(ContextSafeHandle handle, Status status)
             }
             finally
             {
-                Marshal.FreeHGlobal(stringPointer);
+                Marshal.FreeHGlobal(regionPointer);
             }
         }
+
+        void IInternalKmsKeyId.SetAlternateKeyNames(ContextSafeHandle handle, Status status)
+        {
+            this.SetAlternateKeyNames(handle, status);
+        }
```

Comment:
This seems to be an exact duplicate of the public method. Why was it needed?

Reply:
It's a little confusing, but the method calls the extension method which houses the method that is common between AwsKeyId and LocalKeyId